polaris/experimental/_dataset_v2.py

```diff
@@ -40,6 +48,8 @@ class DatasetV2(BaseDataset):
     """
 
     version: ClassVar[Literal[2]] = 2
+    _zarr_md5sum_manifest: str | None = PrivateAttr(None)
+    _zarr_md5sum_manifest_path: str | None = PrivateAttr(None)
```

Comment:
**Suggestion:** Given that the manifest changes significantly between V1 and V2, I think we should remove the manifest from the `BaseDataset` altogether and rather define it directly in the respective subclasses. My bad for introducing it in the base class to begin with.

Reply:
Makes sense, let's do it.